content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1086,6 +1086,23 @@ has been truncated to 1000.
 
 If the number of backend endpoints falls below 1000, the control plane removes this annotation.
 
+### control-plane.alpha.kubernetes.io/leader
+
+Type: Annotation
+
+Example: `control-plane.alpha.kubernetes.io/leader={"holderIdentity":"controller-0","leaseDurationSeconds":15,"acquireTime":"2023-01-19T13:12:57Z","renewTime":"2023-01-19T13:13:54Z","leaderTransitions":1}`
+
+Used on: Endpoints
+
+The {{< glossary_tooltip text="control plane" term_id="control-plane" >}} adds this annotation to
+an [Endpoints](/docs/concepts/services-networking/service/#endpoints) object. This annotation provides
```

Comment:
```suggestion
an [Endpoints](/docs/concepts/services-networking/service/#endpoints) object. This annotation provided
```

Reply:
@kundan2707 In your next PR, please help mark these conversations as "resolved" by clicking the "Resolve conversation" button below when you agree to the suggests and you have revised the PR accordingly.
Checking conversation resolved helps the reviewers confirm that previous discussions are clean, the current reviewer may switch to different aspects of the PR, and a new reviewer wouldn't need to check the whole history of a PR.